pymatviz/io.py

```diff
@@ -135,7 +139,7 @@ def save_fig(
     if isinstance(fig, plt.Figure | plt.Axes):
         if hasattr(fig, "figure"):
             fig = fig.figure  # unwrap Axes
-        fig.savefig(path, **kwargs, transparent=True)
+        fig.savefig(path, **kwargs, transparent=transparent_bg)
```

Comment:
need to merge in case `kwargs` already contains `transparent`

```suggestion
        fig.savefig(path, **kwargs | dict(transparent=transparent_bg))
```